fairlearn/metrics/_group_metric_set.py

```diff
@@ -27,63 +27,71 @@
 _FEATURE_BIN_NAME = 'featureBinName'
 _PREDICTION_TYPE = 'predictionType'
 _PREDICTION_BINARY_CLASSIFICATION = 'binaryClassification'
+_PREDICTION_PROBABILITY = 'probability'
 _PREDICTION_REGRESSION = 'regression'
 _MODEL_NAMES = 'modelNames'
 _SCHEMA = 'schemaType'
 _DASHBOARD_DICTIONARY = 'dashboardDictionary'
 _VERSION = 'schemaVersion'
 
 BINARY_CLASSIFICATION = 'binary_classification'
+PROBABILITY = 'probability'
 REGRESSION = 'regression'
-_allowed_prediction_types = frozenset([BINARY_CLASSIFICATION, REGRESSION])
+_allowed_prediction_types = frozenset([BINARY_CLASSIFICATION, PROBABILITY, REGRESSION])
 
 # The following keys need to match those of _metric_methods in
 # _fairlearn_dashboard.py
 # Issue 269 is about unifying the two sets
-ACCURACY_SCORE_GROUP_SUMMARY = "accuracy_score"
-BALANCED_ROOT_MEAN_SQUARED_ERROR_GROUP_SUMMARY = "balanced_root_mean_squared_error"
-F1_SCORE_GROUP_SUMMARY = "f1_score"
-FALLOUT_RATE_GROUP_SUMMARY = "fallout_rate"
-LOG_LOSS_GROUP_SUMMARY = "log_loss"
-MEAN_ABSOLUTE_ERROR_GROUP_SUMMARY = "mean_absolute_error"
-MEAN_OVERPREDICTION_GROUP_SUMMARY = "overprediction"
-MEAN_PREDICTION_GROUP_SUMMARY = "average"
-MEAN_SQUARED_ERROR_GROUP_SUMMARY = "mean_squared_error"
-MEAN_UNDERPREDICTION_GROUP_SUMMARY = "underprediction"
-MISS_RATE_GROUP_SUMMARY = "miss_rate"
-PRECISION_SCORE_GROUP_SUMMARY = "precision_score"
-R2_SCORE_GROUP_SUMMARY = "r2_score"
-RECALL_SCORE_GROUP_SUMMARY = "recall_score"
-ROC_AUC_SCORE_GROUP_SUMMARY = "balanced_accuracy_score"
-ROOT_MEAN_SQUARED_ERROR_GROUP_SUMMARY = "root_mean_squared_error"
-SELECTION_RATE_GROUP_SUMMARY = "selection_rate"
-SPECIFICITY_SCORE_GROUP_SUMMARY = "specificity_score"
-ZERO_ONE_LOSS_GROUP_SUMMARY = "zero_one_loss"
+ACCURACY_SCORE = "accuracy_score"
+BALANCED_ROOT_MEAN_SQUARED_ERROR = "balanced_root_mean_squared_error"
+F1_SCORE = "f1_score"
+FALLOUT_RATE = "fallout_rate"
+LOG_LOSS = "log_loss"
+MEAN_ABSOLUTE_ERROR = "mean_absolute_error"
+MEAN_OVERPREDICTION = "overprediction"
+MEAN_PREDICTION = "average"
+MEAN_SQUARED_ERROR = "mean_squared_error"
+MEAN_UNDERPREDICTION = "underprediction"
+MISS_RATE = "miss_rate"
+PRECISION_SCORE = "precision_score"
+R2_SCORE = "r2_score"
+RECALL_SCORE = "recall_score"
+ROC_AUC_SCORE = "balanced_accuracy_score"
+ROOT_MEAN_SQUARED_ERROR = "root_mean_squared_error"
+SELECTION_RATE = "selection_rate"
+SPECIFICITY_SCORE = "specificity_score"
+ZERO_ONE_LOSS = "zero_one_loss"
```

Comment:
these changes are not relevant to this PR, right? It's usually easier to review a PR when the scope is limited ;)

Reply:
Strictly speaking, no.